api/src/stampy_chat/followups.py

```diff
@@ -23,6 +23,9 @@ def search_authored(query: str):
 
 
 def get_followups(query):
+    if not query.strip():
+        return []
```

Comment:
this was causing errors when the server returned an error (because of the empty query) which was parsed as followups